api/management/commands/loaddata.py

```diff
@@ -22,22 +23,70 @@ def handle(self, *args, **options):
-                    Disease.objects.create(
+                    disease = Disease(
                         acronym=row['acronym'],
                         name=row['disease']
                     )
+                    disease_list.append(disease)
+                Disease.objects.bulk_create(disease_list)
 
         # Samples
         if Sample.objects.count() == 0:
             sample_path = os.path.join(options['path'], 'samples.tsv')
             with open(sample_path) as sample_file:
                 sample_reader = csv.DictReader(sample_file, delimiter='\t')
+                sample_list = []
                 for row in sample_reader:
                     disease = Disease.objects.get(acronym=row['acronym'])
-                    Sample.objects.create(
+                    sample = Sample(
                         sample_id=row['sample_id'],
                         disease=disease,
                         gender=row['gender'] or None,
                         age_diagnosed=row['age_diagnosed'] or None
                     )
+                    sample_list.append(sample)
+                Sample.objects.bulk_create(sample_list)
+
+        # Genes
+        if Gene.objects.count() == 0:
+            gene_path = os.path.join(options['path'], 'genes.tsv')
+            with open(gene_path) as gene_file:
+                gene_reader = csv.DictReader(gene_file, delimiter='\t')
+                gene_list = []
+                for row in gene_reader:
+                    gene = Gene(
+                        entrezid=row['entrez_gene_id'],
+                        symbol=row['symbol'],
+                        description=row['description'],
+                        chromosome=row['chromosome'] or None,
+                        gene_type=row['gene_type'],
+                        synonyms=row['synonyms'] or None,
+                        aliases=row['aliases'] or None
+                    )
+                    gene_list.append(gene)
+                Gene.objects.bulk_create(gene_list)
+
+        # Mutations
+        if Mutation.objects.count() == 0:
+            mutation_path = os.path.join(options['path'], 'mutation-matrix.tsv.bz2')
+            mutation_df = pd.read_table(mutation_path, index_col=0)
+            mutation_list = []
+            for sample_id, sample_status in mutation_df.iterrows():
+                sample = Sample.objects.get(sample_id=sample_id)
+                mutated_genes = sample_status[sample_status == 1].index
+                for mutated_gene in mutated_genes:
+                    try:
+                        gene = Gene.objects.get(entrezid=mutated_gene)
+                        mutation = Mutation(
+                            gene=gene,
+                            sample=sample
+                        )
+                        mutation_list.append(mutation)
+                    except:
+                        print('OOPS! Had an issue inserting sample', sample_id, 'mutation', mutated_gene)
```

Comment:
This shouldn't happen? Have you had any issues? I don't think we want to except this.